tests/sweep_framework/sweeps/conv2d/short/conv2d_short_sweep.py

```diff
@@ -1583,6 +1583,372 @@
         ],
         "is_conv1d": [True],
     },
+    "tt-forge_sweep_conv2d": {
```

Comment:
can we add one pytest entry to run these locally, similar to what we have for PyTorch traces

Reply:
I am not sure what you mean. could you please elaborate more?

Reply:
Some thing like 
def test_conv2d_localrun(device, input_spec):
that we have for existing sweeps